api/src/main/java/org/openmrs/api/db/hibernate/HibernateUserDAO.java

```diff
@@ -593,9 +598,10 @@ private Query createUserSearchQuery(String name, List<Role> roles, boolean inclu
 			hql.append("inner join user.roles as role ");
 			searchOnRoles = true;
 		}
+		hql.append("where user.uuid != :DAEMON_USER_UUID ");
 		
 		if (!criteria.isEmpty() || searchOnRoles) {
-			hql.append("where ");
```

Comment:
Did you change this accidentally?

Reply:
No @dkayiwa . Could it be error prone?

Reply:
I just wanted to be sure that you were intentional.